### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by mryeminthu**
- Apply React best practices and language style guides in code

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -2,20 +2,23 @@ import React from 'react';
...
+        </li>
+        <li>
+          <Link to="/categories">CATEGORIES</Link>
+        </li>
+      </ul>
+    </div>
+    <div className="navbar-right">
+      <i className="fa-solid fa-user" />
+    </div>
   </nav>
```
Comment (FrederickMih) [inline src/components/Navbar.js:22]: - [ ] Styling should be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/) Font size and weight, colors of the nav items should be the same please.

| Zeplin :+1: | Yours |
| --- | --- |
| Screenshot from 2023-09-29 06-47-28 | Screenshot from 2023-09-29 06-52-06 |

File: src/components/AddBookForm.js [hunk 2]
```diff
@@ -25,34 +25,37 @@ const AddBookForm = () => {
...
+          />
+          <br />
+          <button type="submit" onClick={handleSubmit}>
+            ADD BOOK
+          </button>
+        </form>
+      </div>
+    </>
   );
 };
```
Comment (FrederickMih) [inline src/components/AddBookForm.js]: - [ ] Styling should be **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/) Font size and weight, colors of the headers didn't match that in zeplin. As you can see on the screenshot below only title and category is required here not title and author.

| Zeplin :+1: | Yours |
| --- | --- |
| Screenshot from 2023-09-29 06-54-40 | Screenshot from 2023-09-29 06-54-59 |

File: src/components/Book.js [hunk 3]
```diff
@@ -39,7 +40,7 @@ const Book = ({ id, title, author }) => {
         <section className="new-section3">
           <p>CURRENT CHAPTER</p>
           <h3>Chapter 10</h3>
-          <button type="button">Update Progress</button>
+          <button className="update-button" type="button">Update Progress</button>
         </section>
       </section>
     </article>
```
Comment (FrederickMih) [inline src/components/Book.js]: - [ ] Please once more check the annotated diagram below and fix the changes. The progress bar should be shift a bit to the left, the size of the update bar is a bit larger than expected as you can see below.

| Zeplin :+1:  | Yours |
| --- | --- |
| Bookstore CMS | Screenshot from 2023-09-29 07-18-14 |

File: src/App.css [hunk 4]
```diff
@@ -2,11 +2,68 @@ body {
...
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  border-bottom: 3px solid rgb(173, 170, 170);
+  margin: 20px 30px;
+  padding: 10px 20px;
+  text-decoration: none;
```
Comment (FrederickMih) [inline src/App.css]: - [ ] Please let's make sure the basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices)) No repeating of the same rule. I think you have done this pretty literally in this file. Let's kindly double check file and fix related issues :wrench: it :+1:

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: :raising_hand: @mryeminthu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- App design closer to original designl :+1: 
- Linter checks passed :heavy_check_mark: 


## Required Changes ♻️
**N/A**
_Check the comments under the review._

## Optional suggestions
**N/A**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zeff96 — APPROVED** [review]
### Status: APPROVED
Hi @mryeminthu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights
:heavy_check_mark: Linters passing
:heavy_check_mark: Correct use of GitFlow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**